src/openapi/streaming/streaming.js

```diff
@@ -875,6 +864,69 @@ Streaming.prototype.getQuery = function() {
     }
 };
 
+Streaming.prototype.setOptions = function(
+    options,
+    defaultOptions = DEFAULT_STREAMING_OPTIONS,

```

Comment:
its a weird signature. How about instead where you need to call with the defaultOptions you do this

setOptions({ ...currentOptions, ...newOptions);

Reply:
Done